apps/dashboard/templates/meinberlin_dashboard/includes/project_list_item.html

```diff
@@ -1,30 +1,32 @@
 {% load i18n project_tags %}
 
 <div class="list-item">
-    <div class="list-item__actions">
-        <span>
-            <a href="{% url 'dashboard-project-edit' organisation_slug=view.organisation.slug slug=project.slug %}"
-               class="button">
-                {% trans 'Edit'%}
-            </a>
-        </span>
-    </div>
+    <p class="list-item__subtitle">{{ project.typ }}</p>
     <h3 class="list-item__title">
         <a href="{% url 'project-detail' project.slug %}">
         {{ project.name }}
         </a>
     </h3>
-    <div class="list-item__status">
-        {% if project.has_finished %}
-            {% trans 'Finished' %}
-        {% elif project.active_phase %}
-            {% with project.active_phase.end_date as phase_end %}
-            {% trans 'Active Phase' %}: {{ project.active_phase.content.name }} ({% trans "ends on" %} {{ phase_end }})
-            {% endwith %}
-        {% else %}
-            {% with project.future_phases.first as next_phase %}
-            {% trans 'Next Phase' %}: {{ next_phase.content.name }} ({% trans "starts on" %} {{ next_phase.start_date }})
-            {% endwith %}
-        {% endif %}
+    <div class="list-item__actions">
+        <span class="list-item__action">
+            <a href="{% url 'dashboard-project-edit' organisation_slug=view.organisation.slug slug=project.slug %}"
+               class="button button--light">
+                <i class="fa fa-pencil" aria-hidden="true"></i>
+                {% trans 'Edit'%}
+            </a>
+        </span>
+        <form method="post" class="list-item__action">
+            {% csrf_token %}
+            <input type="hidden" name="project_pk" value="{{ project.pk }}">
+            <button type="submit" name="{{ project.is_draft|yesno:'publish,unpublish' }}"
```

Comment:
I would prefer to have a fixed form field name (e.g. publish) and set the value (e.g. true/false), or is this a common pattern in HTML forms to just use the field name?